### DISCUSSION TITLE
How do I see the underlying api calls to the source?

### DISCUSSION THREAD

**Discussion by meyerovb** [airbytehq/airbyte#34775] [Connector Questions]
Running open source, it's pulling data from a source by calling the source api, does it log anywhere/have anywhere to see the actual http calls (at least the url) being made to the source api?

**Comment by sajarin** [comment]
Airbyte does not expose the underlying API calls to the source in the logs to avoid exposing sensitive information. However, you can try to set the log level to debug to see if you get more information. You can do this by setting LOG_LEVEL=DEBUG. Please note that this might not provide the exact details of the HTTP calls being made to the source API.

Another option is to monitor the network tab in the browser dev tools for one-off checks.

[a comment by an automated account is not shown]
